src/components/CheckList/CheckList.test.tsx

```diff
@@ -371,6 +371,35 @@ test(`Scripted checks appear in the filters when the feature flag is enabled`, a
   expect(within(listBox).getByText(`Scripted`)).toBeInTheDocument();
 });
 
+test(`gRPC checks do not appear in the filters by default`, async () => {
+  const { user } = await renderCheckList();
+  const additionalFilters = await screen.findByText(/Additional filters/i);
+  await user.click(additionalFilters);
+
+  const select = await getSelect({ label: `Filter by type` });
+  await user.click(select[0]);
+  const listBox = screen.getByLabelText(`Select options menu`);
+
+  expect(within(listBox).queryByText(`gRPC`)).not.toBeInTheDocument();
+});
+
+test(`gRPC checks appear in the filters when the feature flag is enabled`, async () => {
```

Comment:
I actually can't get it to show up for me in the filters, even with the feature toggle on

Reply:
When the filter select is opened, `gRPC` is not visible unless the options are scrolled. Is it just hiding further down in the list? 🤷🏻 
![image](https://github.com/grafana/synthetic-monitoring-app/assets/1382208/2855cc97-2311-4139-916a-99549122eda3)
![image](https://github.com/grafana/synthetic-monitoring-app/assets/1382208/03ffb02d-a9e8-477b-a712-8a2372864957)

Reply:
Yup that's 100% what happened